tests/e2e/preferential_topology.go

```diff
@@ -180,6 +184,11 @@ var _ = ginkgo.Describe("[Preferential-Topology] Preferential-Topology-Provision
 		framework.Logf("Waiting for %v for preferred datastore to get refreshed in the environment",
 			preferredDatastoreTimeOutInterval)
 		time.Sleep(preferredDatastoreTimeOutInterval)
+
+		if isSPSServiceStopped {
+			framework.Logf("Bringing sps up before terminating the test")
+			startVCServiceWait4VPs(ctx, vcAddress, spsServiceName, &isSPSServiceStopped)
```

Comment:
after starting  please set  isSPSServiceStopped to appropriate bool value

Reply:
Ack